src/evm_context.rs

```diff
@@ -109,191 +105,160 @@ context_variables! {
     number: Type::Default; None,
     origin: Type::Default; None,
     timestamp: Type::Default; None,
-    memory: Type::Constant("(Array (_ BitVec 256) (_ BitVec 8))"); None,
-    storage: Type::Constant("(Array (_ BitVec 256) (_ BitVec 256))"); None,
-    calldata: Type::Function("((_ BitVec 256))", "(_ BitVec 8)"); None,
-    keccak256_32: Type::Function("((_ BitVec 256))", "(_ BitVec 256)"); None,
-    keccak256_64: Type::Function("((_ BitVec 256) (_ BitVec 256))", "(_ BitVec 256)"); None,
-    keccak256: Type::Function("((Array (_ BitVec 256) (_ BitVec 8)) (_ BitVec 256) (_ BitVec 256))", "(_ BitVec 256)"); None,
-    revert_flag: Type::Default; Some("#x0000000000000000000000000000000000000000000000000000000000000000".to_string()),
-    stop_flag: Type::Default; Some("#x0000000000000000000000000000000000000000000000000000000000000000".to_string())
+    memory: Type::Constant(smt::array(smt::bv(256), smt::bv(8))); None,
+    storage: Type::Constant(smt::array(smt::bv(256), smt::bv(256))); None,
+    calldata: Type::Function(vec![ smt::bv(256) ], smt::bv(8)); None,
```

Comment:
```suggestion
    calldata: Type::Function(vec![smt::bv(256)], smt::bv(8)); None,
```

Reply:
does `fmt` not have an opinion about that?